setup.py

```diff
@@ -51,4 +52,5 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
     ],
+    zip_safe=False,  # for mypy compatibility https://mypy.readthedocs.io/en/latest/installed_packages.html
```

Comment:
This is not just for mypy compatibility. Pyre will not be able to find code inside a zipped install either, so this is needed just as much if anyone wants to have pyre find an installed libcst.